src/orbs/goreleaser.yml

```diff
@@ -112,6 +112,16 @@ jobs:
           source $BASH_ENV
           goreleaser --snapshot --skip-publish --rm-dist
 
+  render-version-schema:
+    executor: default
+    working_directory: /home/ory
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run: |
+          curl -sL https://raw.githubusercontent.com/ory/ci/master/src/scripts/releaser/render-version-schema.sh | bash > $BASH_ENV
+          source $BASH_ENV
```

Comment:
Why are you sourcing the bash script? It does not include any env vars. What you want is this:

```
      -
        run:
          command: "bash <(curl -s https://raw.githubusercontent.com/ory/ci/master/src/scripts/releaser/campaign-send.sh)"
```

Reply:
copy/paste error :sweat_smile: